.python-version

```diff
@@ -1,7 +1,5 @@
-3.7.3
-3.6.8
-3.5.7
-3.4.4
-2.7.16
-2.6.9
+3.8.1
+3.7.6
+3.6.10
+3.5.8
```

Comment:
I think that the latest is 3.5.9 https://www.python.org/downloads/release/python-359/

Reply:
good catch 👍 2be58f3